src/scripts/domain/board.ts

```
@@ -156,6 +156,14 @@ export class Board implements BoardInfo {
     this._board[from.y][from.x] = null;
   }
 
+  public addPiece(piece: Piece) {
+    if (this._board[piece.pos.y][piece.pos.x] !== null) {
+      throw new Error('Cell at ' + piece.pos + ' already occupied');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)